client/client-multi/client-cpp/kaa/EndpointConnectionInfo.hpp

```diff
@@ -27,11 +27,17 @@ struct EndpointConnectionInfo {
     std::string endpointIp_; /**< The ip address of the endpoint. */
     std::string serverIp_; /**< The ip address of the server */
     ServerType serverType_; /**< The type of server, see @c ServerType */
+    /** Server confirmed that credentials are valid and approved the connection on an application level */
+    bool connectionAccepted_;
```

Comment:
Yep, I know. The another way to implement this leads to breaking interfaces. So, I've chosen this way.